### ISSUE TITLE
pem verification file not recognized

### ISSUE THREAD

**Issue by RoxKilly** [schwabe/ics-openvpn#526]
App versions `0.6.47` and `0.6.57` (haven't tested anything in between)
Android `5.1.1`

When importing a `.ovpn` file with a setting like:

```
 crl-verify ca.crl.pem
```

This setting is not recognized so it's appended as a custom configuration with a full path to the `.pem` file. As a result, the file cannot be moved without breaking the setup.  The connection and VPN aren't affected so this may not be an issue for the developer.

The expected behavior: That the file be imported like the `.crt` and password files get imported, so that we may delete the pem file just as we delete the other two after import is successful.

**Comment by schwabe** [comment]
Since commit ff6064fe420c466a266711039fbada851b2832c7 (0.6.50) the file should be embedded when found. Can you share the messages during an import?

**Comment by RoxKilly** [comment]
I mixed up behavior from the two versions because in both cases, deleting/moving the `pem` file would break the setup. Let me be more specific.  All I posted in the OP applies to version `0.6.47`.  

Here is what I experience for `0.6.57` on Android `5.1.1`
1.  On import, the `pem` file is detected and listed under _Certificate Revocation List_
2.  Once the import is successful, if I go into config options, I can see the `pem` listed at the full path to the location where it was found during import
3.  I can connect successfully.
4.  If I remove/rename the `pem`, the connection attempt predictably exists with _error --crl-verify fails with /path/to/pem: No such file or directory_

The expected behavior is what I posted in the OP. Since my observation on both my devices was that I couldn't delete the `pem` after import just as I could delete the other files, I assume it was not recognized properly. Apologies.

**Comment by schwabe** [comment]
Can you make/sahre a screenshot of the import dialog when that happens?

**Comment by schwabe** [comment]
crl-verify files on a mobile device is a strange thing. The idea was that users might update these from time to time, so ics-openvpn tries to refer to their original position instead of importing them into the file. However this seems to bring more confusion. I might change back to import it into the profile

**Comment by RoxKilly** [comment]
@schwabe Interesting. If I understand your logic, it seems you anticipated that users would want to change the `pem` more often than other input files.  Why?

**Comment by schwabe** [comment]
A certificate is valid as it is valid. A typical crl file has only a validity of 7 or 30 days. Every time a CA revokes a certificate the CRL file is updated.

**Comment by RoxKilly** [comment]
@schwabe ok; that was helpful. I bet most users don't know the purpose of this file and won't bother to look into it.  They'll  just get it from the VPN provider and expect things to work.

Off-topic: given the short lifespan of this file, I wonder why it doesn't live on a remote server instead of being locally maintained. I've had the same VPN provider for a long time, and the only time I changed this file was once when the provider lost control of its old keys and sent me a new package with updated certs and connection configs.  Well over 30 days have passed since I opened this issue, so my CRL is likely expired but until I read your last post, I had never given that any thought.
